RestaurantReservation.DB/Models/MenuItem.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.EntityFrameworkCore;
+
+namespace RestaurantReservation.DB.Models;
+
+[PrimaryKey(nameof(ItemId))]
```

Comment:
inconsistency, use the fluent api

Reply:
I think we discussed this in our meeting and we agreed to keep it like that.